client/components/AssignDoi/AssignDoi.js

```diff
@@ -64,7 +70,14 @@ const getButtonText = (state, hasExistingDeposit) => {
 export function reducer(state, action) {
 	switch (action.type) {
 		case AssignDoiActionType.FetchPreview:
-			if (!(state.state === AssignDoiState.Initial || state.state === AssignDoiState.Error)) {
+			if (
+				!(
+					state.state === AssignDoiState.Initial ||
```

Comment:
Definitely no need to change this here, but I find that calling this kind of value a `status` helps distinguish it from the containing `state` object.